cmd/openshift-install/create.go

```diff
@@ -269,40 +269,46 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
 		return errors.Wrap(err, "waiting for Kubernetes API")
 	}
 
-	eventTimeout := 30 * time.Minute
-	logrus.Infof("Waiting up to %v for the bootstrap-complete event...", eventTimeout)
-	return waitForEvent(ctx, client.CoreV1().RESTClient(), "bootstrap-complete", eventTimeout)
+	return waitForBootstrapConfigMap(ctx, client)
 }
 
-// waitForEvent watches the events in the kube-system namespace, waits
-// for the event of the given name, and prints out all other events on
-// the way.
-func waitForEvent(ctx context.Context, client cache.Getter, name string, timeout time.Duration) error {
+// waitForBootstrapConfigMap watches the configmaps in the openshift namespace
+// and waits for the bootstrap configmap to report that bootstrapping has
+// completed.
+func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset) error {
+	timeout := 30 * time.Minute
+	logrus.Infof("Waiting up to %v for bootstrapping to complete...", timeout)
+
 	waitCtx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
-	resource := "events"
-	namespace := "kube-system"
-
 	_, err := clientwatch.UntilWithSync(
 		waitCtx,
-		cache.NewListWatchFromClient(client, resource, namespace, fields.Everything()),
-		&corev1.Event{},
+		cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", "openshift", fields.OneTermEqualSelector("metadata.name", "bootstrap")),
```

Comment:
why did we move from kube-system to openshift namespace ?

Reply:
3118a6185...1a3cc8be6